Gopkg.toml

```diff
@@ -21,13 +21,17 @@
 #  version = "2.4.0"
 
 
+[[constraint]]
+  name = "github.com/aws/amazon-ecs-agent"
```

Comment:
Why did you vendor the ECS agent?

Reply:
Right now, it uses some code from amazon-ecs-agent and amazon-ecs-cni-plugins. We are planning to move these shared code out into another library repo and only vendor this new library repo.

Reply:
What code are you using?  It's worth evaluating whether to factor it out or to just stop using the code; not everything in the ECS agent is code that we'd recommend reusing.

Reply:
opened issue #27 to track this.   Thanks